ttnn/cpp/ttnn/operations/eltwise/binary/binary.cpp

```diff
@@ -140,7 +140,7 @@ Tensor BinaryOperation<binary_op_type, in_place>::invoke(
     repeat_smaller(input_tensor_a, input_tensor_b);
     repeat_smaller(input_tensor_b, input_tensor_a);
 
-    return ttnn::prim::binary(
+    ttnn::Tensor result = ttnn::prim::binary(
         queue_id,
```

Comment:
Why assign this to a variable and not just return it?

Reply:
Looks like some debugging artifact 😁

Reply:
easier to debug. Because it will become local_tensor in launch_op.

Reply:
Please remove this change